kbase-extension/static/kbase/js/widgets/function_output/kbaseReportView.js

```diff
@@ -75,48 +82,49 @@ define(
         },
         reportData: null,
 
-        //this is an ugly hack. It'd be prettier to hand in just the shock node ID, but I don't have one of those yet.
-        //Also, this is embedding the token into the html and the URL, both of which are potentially security concerns.
-        importExportLink : function(shock_url, name) {
-          var m;
-          if (m = shock_url.match(/\/node\/(.+)$/)) {
-            var shock_id = m[1];
-            var url = "https://ci.kbase.us/services/data_import_export/download?&id=" + shock_id + "&token=" + this.authToken() + "&wszip=0&name=" + name;
-
-            return url;
-          }
+        // this is an ugly hack. It'd be prettier to hand in just the shock node ID, but I don't have one of those yet.
+        // Also, this is embedding the token into the html and the URL, both of which are potentially security concerns.
+        // TODO: update to put the auth token into the url... should be working in CI already.
+        // TODO: NO HARDCODING OF URLS!!!
+        importExportLink: function (shock_url, name) {
+            var m = shock_url.match(/\/node\/(.+)$/);
+            if (m) {
+                var shock_id = m[1];
+                var url = 'https://ci.kbase.us/services/data_import_export/download?&id=' + shock_id + '&token=' + this.authToken() + '&wszip=0&name=' + name;
```

Comment:
This should be a configure option in src/config.json. At least the URL - the form params should just stay.